docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
 # Connect an æpp to a wallet
 
-## Introduction
+This guide describes the 4 steps that are necessary to connect your application to a wallet using the RPC API.
```

Comment:
some of the guides does not have this `Introduction` heading. Its kinda inconsistent. I think it can go without heading